app/lib/filters/checkbox_filter.rb

```diff
@@ -0,0 +1,12 @@
+module Filters
+  class CheckboxFilter < Filter
+    def value
+      # 'params' for a checkbox should be true or false. the value
+      # we send to rummager can be params or a static value provided in
+      # the finder content item
+      if params
+        facet['filter_value'] || params
+      end
+    end
```

Comment:
The filters have all been moved (without changes) into individual class files for readability.

Reply:
👍